web/routes/web.php

```diff
@@ -11,10 +11,18 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+use Illuminate\Support\Facades\Auth;
 
+// Autogenerated routes for authentification.
 Auth::routes();
 
-Route::get('/home', 'HomeController@index')->name('home');
```

Comment:
Auth redirects to /home upon user registration.
Consider this:
```php
Route::get('/home', function () {
    return redirect('/');
});
```
or something similar.